tests/test_postgresql.py

```diff
@@ -43,6 +42,7 @@ def test_two_postgreses(postgresql, postgresql2):
 
 def test_rand_postgres_port(postgresql_rand):
     """Check if postgres fixture can be started on random port."""
+    import psycopg2
```

Comment:
This shouldn't need moving since the test environment always has `psycopg2` installed?

Reply:
right, moved it just to properly see the error message locally ;)